### PR TITLE
Please review my code

### PR DESCRIPTION
**Pull request by shahzaman900**
### please review my code

### CODE REVIEW

**Review by noelfoka — COMMENTED** [review]
Great work, what you have done as work is extraordinary and deserves encouragement.. 

However, there are a few modifications to be made and I suggest that you follow the following instructions in order for your work to be approved:

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+}
+
+function handleKeyPress(e) {
+  const inputValue = document.getElementById('input');
+  const description = inputValue.value;
+  if (e.key === 'Enter') {
+    e.preventDefault();
+    addTask(description);
+    inputValue.value = '';
+  }
```
Comment (noelfoka) [inline src/crud.js:21]: Write each function in a specific file that best describes the functionality. do this for your crud.js and index.js files, so your project is clean and clear 👍🏼.

File: src/modules/completedTask.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line
```
Comment (noelfoka) [inline src/modules/completedTask.js:1]: delete this comment and solve the problem according to the configurations of Linters and it will be perfect.

**Review by noelfoka — CHANGES_REQUESTED** [review]
After these changes, resubmit your project again. I wish you excellent coding for the rest and for any concerns, you can write to me directly by this comment.

File: src/modules/completedTask.js (hunk 2 above)
Comment (shahzaman900) [reply src/modules/completedTask.js:1]: thank you so much for highlighting the issues I will resolve these issues thank you.

**Pull request merged**